### PR TITLE
Fix(eos_cli_config_gen): Remove requirement for original_ip to be unique under interface IP NAT

### LINKED ISSUE #2848: Feature: Add support for Postcard telemetry
### Enhancement summary

At the moment, if we want to use Postcard telemetry, we need to add the eos_cli commands. Ideally this would be supported in the `eos_cli_config_gen` data model.

Documentation about the EOS feature can be found here : 
https://www.arista.com/en/um-eos/eos-sampled-flow-tracking#unique_933198740

### Which component of AVD is impacted

eos_cli_config_gen

### Use case example

Using Postcard, also known as GREEN-T telemetry on AVD managed devices.

### Describe the solution you would like

Schema and templates extended to support Postcard.

### Describe alternatives you have considered

Using `eos_cli` with the whole command line commands. This is not optimal.

### Additional context

_No response_

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #3894: Feat(eos_cli_config_gen): Interface IP NAT Support - origin_ip is not unique
### Issue Summary

# Issue
We are using the IP NAT feature implemented in https://github.com/aristanetworks/avd/pull/2750 and have the issue that our Ansible Playbook Log gets flushed with error messages stating:

`Validation Error: svi_profiles[26].structured_config.ip_nat.destination.static': Values of Primary key 'original_ip' are not unique as required.`

The reason seems to be that `original_ip` is marked as `unique` in ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/data_model/Interfaces.md. 

The `original_ip` does not have to be unique, because the combination of `original_ip` and `original_port` has to be unique. The same would be true for `translated_ip` and `translated_port`, which are both not marked as `unique`. 

Please unmark `original_ip` as `unique` or make sure that only the combination of `original_ip` and `original_port` is validated to be unique. This makes the creation of multiple entries for one `original_ip` and different `original_ports` possible without creating misleading warnings.

### Which component(s) of AVD impacted

eos_cli_config_gen

### How do you run AVD ?

Ansible CLI (with virtual-env or native python)

### Steps to reproduce

```shell
1. Create SVI Profile or Interface Configuration with IP NAT and multiple Entries with the same "original_ip". 
2. Execute Playbook and observe Warnings
3. See that configuration gets generated and can be implemented and works in EOS.
```

### Relevant log output

```shell
'Validation Error: svi_profiles[26].structured_config.ip_nat.destination.static': Values of Primary key 'original_ip' are                                                                                                                                                                   not unique as required.
```

### Contributing Guide

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by laxmikantchintakindi**
## Change Summary

original_ip need not be unique in interface_ip_nat_static. 

## Related Issue(s)

Fixes #3894

## Component(s) name

`arista.avd.eos_cli_config_gen`

## Proposed changes
Remove original_ip as primary key from interface_ip_nat_static.

## How to test
Add an element in interface_ip_nat_static with same original_ip and different original_ports.

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_interface_ip_nat_static.schema.yml [hunk 1]
```diff
@@ -35,6 +34,7 @@ $defs:
           description: IPv4 address.
         original_port:
           type: int
+          description: The combination of original_ip and original_port must be unique.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_interface_ip_nat_static.schema.yml]: Maybe also add this for the original_ip

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_interface_ip_nat_static.schema.yml]: Done. Added to translated_ip and translated_port too.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ip-nat.md [hunk 2]
```diff
@@ -116,16 +116,18 @@
...
                 original_port: <int; 1-65535>
                 priority: <int; 0-4294967295>
                 protocol: <str; "udp" | "tcp">
 
-                # IPv4 address.
+                # IPv4 address. The combination of translated_ip and translated_ports must be unique.
                 translated_ip: <str; required>
 
-                # requires 'original_port'.
+                # requires 'original_port'. The combination of translated_ip and translated_ports must be unique.
```
Comment (MaheshGSLAB) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ip-nat.md]: we dont need translated ip and translated port combination to be unique. Its applicable for original ip and port
```
ip nat profile Test1
   ip nat destination static 10.1.1.1 10 access-list Test1 10.1.1.2 20
   ip nat destination static 10.1.1.1 20 access-list Test1 10.1.1.2 20
```

Comment (laxmikantchintakindi) [reply ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/ip-nat.md]: Corrected.

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_interface_ip_nat_static.schema.yml [hunk 3]
```diff
@@ -32,9 +31,10 @@ $defs:
           description: "'access_list' and 'group' are mutual exclusive."
         original_ip:
           type: str
-          description: IPv4 address.
+          description: IPv4 address. The combination of original_ip and original_port must be unique.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_interface_ip_nat_static.schema.yml]:
```suggestion
          description: IPv4 address. The combination of `original_ip` and `original_port` must be unique.
```

File: ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_interface_ip_nat_static.schema.yml [hunk 4]
```diff
@@ -32,9 +31,10 @@ $defs:
...
         original_port:
           type: int
+          description: The combination of original_ip and original_port must be unique.
```
Comment (ClausHolbechArista) [inline ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/defs_interface_ip_nat_static.schema.yml]:
```suggestion
          description: TCP/UDP port. The combination of `original_ip` and `original_port` must be unique.
```

**Comment by ClausHolbechArista** [comment]
pre-commit.ci autofix

**Review by ClausHolbechArista — APPROVED** [review]

**Review by gmuloc — APPROVED** [review]

**Pull request merged**